bin/serve

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
+
+const Express = require('express');
+const ExpressHandlebars = require('express-handlebars');
+const BodyParser = require('body-parser');
+const path = require('path');
+const http = require('http');
+const apiApp = require('../api/app');
+
+const apiService = require('../src/services/api');
+const config = require('../src/config.json');
+
+
+// create static app
+const urlencoded = BodyParser.urlencoded({
+  extended: false,
+});
+const staticApp = Express();
```

Comment:
Let's not have two apps on two ports.
Better that we have one that serves both.

Reply:
they are loaded into one main app